components/post/PostImage.tsx

```diff
@@ -0,0 +1,11 @@
+import Image from 'next/image';
+import styles from './Post.module.scss';
+
+function PostImage({ image, title }: { image: string; title: string }) {
```

Comment:
```suggestion
function PostImage({ imageUrl, title }: { imageUrl: string; title: string }) {
```
그냥 image라고 하면 url이나 uri인지 혹은 컴포넌트인지 파일인지 알수가없을것같아요!